aten/src/ATen/native/mps/OperationUtils.h

```diff
@@ -5,6 +5,7 @@
 #include <ATen/Utils.h>
 #include <ATen/mps/MPSStream.h>
 #include <ATen/native/mps/TensorFactory.h>
+#include <ATen/native/mps/MPSGraphVenturaOps.h>
```

Comment:
Looks good, but it would've been better to add this header to the specific units that need it, to not compile it in every .mm file.

Reply:
Thanks - I have updated it